http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java

```diff
@@ -289,4 +291,30 @@ public HttpClientConfig setKerberosRemoteServiceName(String serviceName)
         this.kerberosRemoteServiceName = serviceName;
         return this;
     }
+
+    public DataSize getHttp2InitialSessionReceiveWindowSize()
```

Comment:
Add `@NotNull` and min/max. Thoughts on min/max values? Probably 1GB for max. Maybe 1kB for min?